hie.yaml

```diff
@@ -0,0 +1,2 @@
+cradle:
```

Comment:
Huh, what's `cradle`?

Reply:
Some abstraction of haskell-language-server for how to load a project.
https://github.com/haskell/hie-bios
I need it this here to instruct HLS to load the project with Cabal instead of Stack, iirc because the Stack support has an issue with multi-library packages.